core/modules/user/views/user.views.inc

```diff
@@ -268,6 +268,79 @@ function user_views_data() {
     ),
   );
 
+  // Changed field.
+  $data['users']['changed'] = array(
+    // The item it appears as on the UI.
+    'title' => t('Changed date'),
+    // The help that appears on the UI.
+    'help' => t('The date the user was changed.'),
+    'field' => array(
+      'handler' => 'views_handler_field_date',
+      'click sortable' => TRUE,
+    ),
+    'sort' => array(
+      'handler' => 'views_handler_sort_date',
+    ),
+    'filter' => array(
+      'handler' => 'views_handler_filter_date',
+    ),
+  );
+
+  $data['users']['changed_fulldate'] = array(
+    'title' => t('Changed date'),
+    'help' => t('Date in the form of CCYYMMDD.'),
+    'argument' => array(
+      'field' => 'changed',
+      'handler' => 'views_handler_argument_node_created_fulldate',
+    ),
+  );
+
+  $data['users']['changed_year_month'] = array(
+    'title' => t('Changed year + month'),
+    'help' => t('Date in the form of YYYYMM.'),
+    'argument' => array(
+      'field' => 'changed',
+      'handler' => 'views_handler_argument_node_created_year_month',
+    ),
+  );
+
+  $data['users']['timestamp_year']['moved to'] = array('users', 'changed_year');
```

Comment:
I believe this line here is incorrect and will cause issues.  This was probably adapted from `$data['users']['timestamp_year']['moved to'] = array('users', 'created_year');`

What the original line is telling the views controller is that, in old times, there was a views field/filter called `timestamp_year` that doesn't exist anymore. So, before this PR line above, the original line meant "Dear Views: if you find an old views field/filter called `timestamp_year`, please use `created_year` instead."

The problem with the line in this PR is that it's telling views to look for `changed_year` instead, and this is not correct. It should look for `created_year`, since that's the field that replaced `timestamp_year`.

Reply:
I hadn't noticed this line amongst the ones I copied and adapted from created. I agree it can be removed.